index.js

```diff
@@ -226,7 +226,7 @@ const ScrollableTabView = createReactClass({
     });
 
     if (this.contentScrollDistance >= this.collapsableBarHeight) {
-      this.contentView.scrollTo({ x: 0, y: this.collapsableBarHeight, animated: false });
+      this.contentView.getNode().scrollToOffset({ offset: this.collapsableBarHeight, animated: false });
```

Comment:
do we need a check for whether we have a collapsible element or not?

Reply:
I was thinking that - but I think we're actually protected by the fact that if we don't have a collapsible element, then `collapsableBarHeight` will be undefined. 

And this doesn't cause any issues on the standings/match calendar - where we don't have a collapsible element